docs/docs/cmd/request.md

```diff
@@ -0,0 +1,64 @@
+# request
+
+Executes the specified web request using CLI for Microsoft 365
+
+## Usage
+
+```sh
+m365 request [options]
+```
+
+## Options
+
+`-u, --url <url>`
+: The request URL. 
+
+`-m, --method [method]`
+: The HTTP request method. Accepted values are `get, post, put, patch, delete, head, options`. The default value is `get`.
+
+`-r, --resource [resource]`
+: The resource uri for which the CLI should acquire a token from AAD in order to access 
+the service.
+
+`-b, --body [body]`
+: The request body. Optionally use `@example.json` to load the body from a file. 
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+The request will be issued as bare as possible, meaning you are responsible for most request information, such as headers, method and body. There are a few exceptions: 
+
+- the command does apply compression and throttling handling as part of the request execution. 
+- The `accept` header is set and defaults to `application/json`. 
```

Comment:
Here, we need to make it clear that you can set the `accept` header but if you don't, it defaults to `application/json`. This is different than the previous point, where you have no ability to control whether you want to use compression and throttling handling or not.

Reply:
clearer now?